kili/cli/project/import_.py

```diff
@@ -21,12 +21,10 @@
 
 def type_check_asset(key, value):
     """type check value based on key"""
-    if (
-        key == "content"
-        and not os.path.isfile(value)
-        and not urllib.request.urlopen(value).getcode() == 200
-    ):
-        return f"{value} is not a valid url or path to a file."
+    if key == "content" and not os.path.isfile(value):
```

Comment:
methods should start with a verb: check_type_asset.
Btw, I do not see why this is called a type check, this is more a general asset validation right ?

Reply:
I changed the name to check_asset_type for the moment.
I didn't write this function, Florian did, I just had to do a change inside because it raised a pylint error. And I admit that the name if the function is not that good